todo/src/App.js

```diff
@@ -1,27 +1,54 @@
-// import logo from './logo.svg';
-import {useState} from 'react';
-
-import logo from './tek_logo.png';
+import React, {useState} from 'react';
 import './App.css';
 import Todo from './Components/Todo';
+import TodoForm from './Components/TodoForm';
+
 
 function App() {
 
-  const [todos, setTodos] = useState(["eat","sleep","code","pray","repeat"]);  //sample todo items.  these will need to be changed in your app.  Just filler todos, although they are quite important!
+  const [todos, setTodos] = useState([
+    {
+      text: '',
+      isCompleted: false
+    }
+  ]);
 
+  const addTodo = text => {
```

Comment:
All functions that are being used from one component shouldn't be in the App component. Like, addTodo should be in the TodoForm.